cli/src/commands/application/instances/connect.rs

```diff
@@ -271,6 +275,33 @@ pub async fn handle_connect(
         );
         eprintln!();
 
+        match (
+            &instance_object.node_name,
+            &instance_object.pod_ip,
+            &instance_object.cookie,
+        ) {
+            (Some(node_name), Some(pod_ip), Some(cookie)) => {
+                eprintln!(
+                    "To connect to mv-wukong-api-proxy, start a new Notebook and click on `+Smart`\nand select `Remote Execution`. Follow the instructions that pop up to restart the Notebook.\nEnter these values into the `NODE` and `COOKIE` input boxes:"
```

Comment:
I'm think what about making it more structure? like 

```
To connect to mv-wukong-api-proxy:
1. start a new Notebook
2. click on `+Smart`
3. select `Remote Execution`. 
4. Follow the instructions that pop up to restart the Notebook.
5. Enter these values into the `NODE` and `COOKIE` input boxes:
    NODE: 🖥️: ...
    COOKIE: 🍪: ...
```

Reply:
I like this. So much clearer. Lemme update.